### PR TITLE
Move all apihelp messages to separate files

### PR DESCRIPTION
**Pull request by amire80**
This has already been done to many MediaWiki extensions
on Gerrit, and at least one on GitLab.

This is done to make translation more convenient by separating
the more technical API messages from the UI messages.

The corresponding translatewiki configuration change is at
https://gerrit.wikimedia.org/r/c/translatewiki/+/697081

Bug: T189982

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -11,14 +11,6 @@
...
 	"special-recordWizard-intro": "",
 
-	"apihelp-query+rwrecords-summary": "List all transcriptions of records made by a speaker in a given language.",
-	"apihelp-query+rwrecords-param-language": "Qid of the language's local wikibase item.",
-	"apihelp-query+rwrecords-param-speaker": "Qid of the speaker's local wikibase item.",
-	"apihelp-query+rwrecords-param-sort": "Property to sort by.",
-	"apihelp-query+rwrecords-param-dir": "Direction of the sort.",
-	"apihelp-query+rwrecords-param-format": "Element to get as output.",
-	"apihelp-query+rwrecords-param-limit": "How many transcriptions in total to return.",
-
```
Comment (Poslovitch) [inline i18n/en.json:21]: Shouldn't these be moved to a file named `i18n/api/en.json`?

File: i18n/qqq.json [hunk 2]
```diff
@@ -17,13 +17,6 @@
 	"recordWizard-desc": "{{desc|name=RecordWizard|url=https://www.mediawiki.org/wiki/Extension:RecorderWizard}}",
 	"special-recordWizard-title": "Title of the special page Special:RecordWizard",
 	"special-recordWizard-intro": "Description appearing on top of RecordWizard.",
-	"apihelp-query+rwrecords-summary": "{{doc-apihelp-summary|query+rwrecords}}",
-	"apihelp-query+rwrecords-param-language": "{{doc-apihelp-param|query+rwrecords|language}}",
-	"apihelp-query+rwrecords-param-speaker": "{{doc-apihelp-param|query+rwrecords|speaker}}",
-	"apihelp-query+rwrecords-param-sort": "{{doc-apihelp-param|query+rwrecords|sort}}",
-	"apihelp-query+rwrecords-param-dir": "{{doc-apihelp-param|query+rwrecords|dir}}",
-	"apihelp-query+rwrecords-param-format": "{{doc-apihelp-param|query+rwrecords|format}}",
-	"apihelp-query+rwrecords-param-limit": "{{doc-apihelp-param|query+rwrecords|limit}}",
```
Comment (Poslovitch) [inline i18n/qqq.json:26]: Same thing here (but `i18n/api/qqq.json`).

**Review by Poslovitch — CHANGES_REQUESTED** [review]

File: i18n/en.json (hunk 1 above)
Comment (amire80) [reply i18n/en.json:21]: Drat! I created the files, but forgot to add them. Fixing...

Comment (amire80) [reply i18n/en.json:21]: Should be fixed, please check.

**Review by Poslovitch — APPROVED** [review]
This looks all good now. Thanks.

**Pull request merged**